src/models/__tests__/CategoriesMapModel.spec.js

```diff
@@ -81,6 +81,10 @@ describe('CategoriesMapModel', () => {
   const category3 = categoriesMapModel.findCategoryByPath('/augsburg/de/erste-schritte///')
   const category4 = categoriesMapModel.findCategoryByPath('/augsburg/de/erste-schritte/asylantrag')
 
+  if (!category1 || !category2 || !category3 || !category4) {
```

Comment:
when does that happen?

Reply:
With the test setup as it is, never. However, it is required for flow types since `findCategoryByPath` may return `undefined | null` if the category is not found.